frontend-ssr/components/cli_git/git_commands_cc.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+
+interface Props {
+  defaultValue:Array<RegExp> | Array<string>
+}
+interface State {
+  currentCommandIndex: number,
+  message: string,
+  buttonText: string
+}
+
+export default class MatchCommandClassComponent extends React.Component<Props, State> {
+  constructor(props) {
+    super(props);
+
+    this.handleSubmit = this.handleSubmit.bind(this);
```

Comment:
We can skip this step with the help of `babel` which is built-into this project. Just change the function declaration on line 25 to this:

```
handleSubmit  = (event) => { 
  // ...
}
```